src/js/modules/displayTodos.js

```diff
@@ -0,0 +1,56 @@
+import deleteImg from '../../img/delete.svg';
+import dragImg from '../../img/drag.svg';
+import checkBtnImg from '../../img/done.svg';
+
+const displayTodos = (list) => {
+  /* reset the inner content of the todo task list */
```

Comment:
- [ ] `optional`
- [ ] Kindly remove the commented lines everywhere cause its making your code look messy, please read this  [article](https://www.parkersoftware.com/blog/how-to-comment-code-meaningfully-a-best-practice-guide/#:~:text=There's%20no%20value%20in%20commenting,information%20have%20a%20negative%20value.) To know how to comment minus making your code look messy 🥂